SMLHelper/Options/SliderModOption.cs

```diff
@@ -6,6 +6,9 @@
     using UnityEngine;
     using UnityEngine.UI;
     using UnityEngine.Events;
+#if BELOWZERO
```

Comment:
Can it be something like this?
```cs
#if SUBNAUTICA
    using Text = UnityEngine.UI.Text;
#endif
#if BELOWZERO
    using Text = TMPro.TextMeshProUGUI;
#endif
```

Reply:
This should also work. Personally I don't have a preference for either style.

Reply:
yup this works as well. and as I have no issue either way I put that in.

Reply:
Less duplicated code this way